### ISSUE TITLE
adorn_totals fails after non-piped tabyl() call on a vector

### ISSUE THREAD

**Issue by sfirke** [sfirke/janitor#118]
Good:
```
mtcars %>% tabyl(cyl) %>% adorn_totals("row")
    cyl  n percent
1     4 11 0.34375
2     6  7 0.21875
3     8 14 0.43750
4 Total 32 1.00000
```

Very bad:
```
> tabyl(mtcars$cyl) %>% adorn_totals("row")
Error in eval(x$expr, data, x$env) : object 'mtcars' not found
```

Look at whether the dollar sign in the name of column 1 is the problem?

**Comment by sfirke** [comment]
This fails too: `crosstab(mtcars$cyl, mtcars$carb) %>% adorn_totals()`

**Comment by sfirke** [comment]
removed an entirely redundant line of code 🤔 (looks like this problem is fixed under dplyr 0.6.0, so users of janitor 0.3.0 on CRAN can fix by upgrading their dplyr if they don't want the latest janitor from GitHub)
